test/parallel/test-tls-over-http-tunnel.js

```diff
@@ -145,29 +148,29 @@ proxy.listen(0, function() {
       socket: socket,  // reuse the socket
       agent: false,
       rejectUnauthorized: false
-    }, function(res) {
+    }, (res) => {
       assert.strictEqual(200, res.statusCode);
 
-      res.on('data', function(chunk) {
+      res.on('data', common.mustCall((chunk) => {
```

Comment:
Strictly speaking, is there a guarantee that the data won't be broken up into more than one chunk (resulting in the event being emitted more than one time and the callback running more than one time)?

If "yes", then `gotRequest` (and the `exit` handler) can be removed from the test.

Reply:
Oh, wait, I'm wrong about `gotRequest` being removable unless we can also wrap the arrow function declared on line 151 in `common.mustCall()`.

Reply:
(And `onConnect` as well would need to be wrapped. I should have been more careful before leaving that first comment about line 154. Maybe best just to ignore all three of these comments. :-D )

Reply:
Deleting the comments because they're noise. :-D